provider/common/bootstrap_test.go

```diff
@@ -135,6 +135,55 @@ func (s *BootstrapSuite) TestCannotStartInstance(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "cannot start bootstrap instance: meh, not started")
 }
 
+func (s *BootstrapSuite) TestBootstrapSeries(c *gc.C) {
+	s.PatchValue(&jujuversion.Current, coretesting.FakeVersionNumber)
+	s.PatchValue(&series.HostSeries, func() string { return "precise" })
```

Comment:
I'm going to be that person and say -- even though we don't have time to fix this right now -- patching values is a smell that our design is bad, and we should stop doing this in tests.

Carry on! ;)